user.go

```diff
@@ -24,3 +27,21 @@ func (u *User) String() string {
 func (u *User) Mention() string {
 	return fmt.Sprintf("<@%s>", u.ID)
 }
+
+// AvatarURL returns a URL to the user's avatar.
+//		size:     The size of the user's avatar as a power of two
+func (u *User) AvatarURL(size string) string {
+	return UserAvatarURL(u.ID, u.Avatar, size)
+}
+
+// UserAvatarURL returns a URL to the requested user's avatar
+//		userID:   The ID of the user to get
+//		avatarID: The ID of the user's avatar
+//		size:     The size of the user's avatar as a power of two
+func UserAvatarURL(userID, avatarID, size string) string {
+	extension := ".jpg"
+	if strings.HasPrefix(avatarID, "a_") {
+		extension = ".gif"
+	}
+	return EndpointCDNAvatars + userID + "/" + avatarID + extension + "?size=" + size
```

Comment:
I noticed that the `EndpointUserAvatar` function requests a png image. Does the file format change to a png when you request it as one?